src/chapter1/section1/Exercise33_Matrix_Operations.java

```diff
@@ -65,12 +65,11 @@ public static void main(String[] args) {
 		
 		StdOut.println();
 		//Vector-matrix product
-		double[] i = {1, 2, 3};
```

Comment:
Why is this element being deleted?

Reply:
In the example, we used a 3x3 matrix and a vector which have 3 elements, so we can do both matrix-vector product or vector-matrix product.
If we use  a 2x3 matrix, then the vector must have 2 elements  so that we can perform vector-matrix product.
I deleted this element to make the example clearer, avoiding confusion about the order of matrix and vector in multiplication as below
```
for (int i = 0; i < a.length; i++) {
	for (int j = 0; j < a[0].length; j++) {
```